tests/unit/basic_crawler/test_basic_crawler.py

```diff
@@ -734,6 +734,40 @@ async def handler(context: BasicCrawlingContext) -> None:
     assert stats.requests_finished == 1
 
 
+@pytest.mark.parametrize(
+    ('n_requests', 'n_error_request', 'e_starts', 'e_finished'),
```

Comment:
Could you rename these so that it's more obvious what they do? The seconds one is the most hard one to understand for me.